src/coreclr/scripts/superpmi_aspnet.py

```diff
@@ -151,7 +151,7 @@ def remove_readonly(func, path, _):
         print ("Executing in " + temp_location)
 
         # install dotnet 6.0
```

Comment:
```suggestion
        # install dotnet 7.0
```

Reply:
Thanks for spotting this. Given that it's just a comment in a script that apparently only I ever use, I'll try and fix it next time I need to modify this.